crates/functional/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "yew-functional"
+version = "0.13.0"
+authors = ["ZainlessBrombie <zainredirect@gmail.com>"]
+edition = "2018"
+repository = "https://github.com/yewstack/yew"
+homepage = "https://github.com/yewstack/yew"
+license = "MIT/Apache-2.0"
+keywords = ["web", "wasm", "frontend", "webasm", "webassembly"]
+categories = ["gui", "web-programming", "wasm"]
+description = "A framework for making client-side single-page apps"
+
+
+[badges]
+travis-ci = { repository = "yewstack/yew" }
+
+[dev-dependencies]
+wasm-bindgen-test = "0.3.9"
+
+[dependencies]
+yew = { path = "../.." }
+web-sys = { version = "0.3.36", optional = true }
+stdweb = { version = "0.4.20", optional = true }
```

Comment:
can these by `dev-dependencies`?

Reply:
Don't think so? Dev dependencies cannot be optional but feature flags require that

Reply:
Ah I see, maybe we can just run tests using `web-sys` then. It doesn't really make sense for `yew-functional` to have these features.

Reply:
agreed

Reply:
It does need to depend on yew though, right? Which would require it to have one feature flag chosen unless I'm mistaken

Reply:
Anyone using `yew-functional` will be using `yew` which means they can choose which feature when depending on `yew`. Then, cargo will apply that feature globally.

Reply:
Done